doc/arkode/guide/source/sundials/index.rst

```diff
@@ -16,11 +16,15 @@
 Using SUNDIALS
 **************
 
-As discussed in :numref:`ARKODE.Organization`, the six solvers packages
-(CVODE(S), IDA(S), ARKODE, KINSOL) that make up SUNDIALS are built upon common
-classes/modules for vectors, matrices, and algebraic solvers. In addition, the
-six packages all leverage some other common infrastructure, which we discuss
-in this section.
+.. _Organization.Sundials.HighLevelDiagram:
```

Comment:
After previewing the docs, I think it might be better to have the figure at the bottom of the page.